resources/public/api/conf/1.0/application.yaml

```
@@ -39,6 +39,8 @@ paths:
     $ref: "./checkpoints_create.yaml"
   /individuals/self-assesment-test-support/vendor-state/checkpoints/{checkpointId}:
     $ref: "./checkpoints_delete.yaml"
+  /individuals/self-assesment-test-support/vendor-state/checkpoints?nino={nino}:
```

To be consistent with the docs for "Create Checkpoint", I think we need to remove the nino query param from this url (but of course still keep the query param section in "./checkpoints_list.yaml")
```suggestion
  /individuals/self-assesment-test-support/vendor-state/checkpoints:
```